examples/aoc2024/grid.jou

```diff
@@ -0,0 +1,112 @@
+# This file contains a utility class for AoC solutions.
```

Comment:
File shouldn't be copied from 2023, should instead import from there or move the file